### ISSUE TITLE
award recommendation: when you get no exact hits, your input shoul carry over into the next page

### ISSUE THREAD

**Issue by SaraUckelman** [uckelman/dw_op#22]
Suppose I want to recommend 'fdsfda" for an award, so I put this into the "search" field.  There are no exact matches, so I am returned an empty text box into which a name can be entered free hand.  That box should come pre-populated with what I've already entered: "fdsfda".  Then I can edit that, or simply hit "submit".

**Comment by MariaLaine** [comment]
I agree with this suggestion. Additionally, a text explaining that after the search producing no exact matches, one can select the empty spot and write the name there, should be added. Especially when the search produces a bunch of "near matches", the user might not even scroll down far enough to see the empty box.
